.github/workflows/docker-publish.yml

```diff
@@ -7,21 +7,6 @@ name: Docker
 
 on:
   workflow_dispatch:
-  schedule:
-    # Build the job every day for the master branch
-    - cron: '0 0 * * *'
-  push:
-    branches: [ "main", 'v*.*.*' ]
-    tags: [ 'v*.*.*' ]
-  pull_request:
-    branches: [ "main", 'v*.*.*' ]
```

Comment:
Just a thought if we could manually build the images on demand through `workflow_dispatch`. 

This will keep multiple branches in the parent repository to have just the k8s version mentioned for a new stable release of k8s.

Reply:
Not applicable anymore.